tmt.spec

```diff
@@ -90,6 +90,20 @@ Recommends:     qemu-system-x86-core
 
 %description -n tmt+provision-virtual %_metapackage_description
 
+%package -n     tmt+provision-bootc
+Summary:        Dependencies required for tmt bootc machine provisioner
+Provides:       tmt-provision-bootc == %{version}-%{release}
+%if 0%{?fedora} < 40
+Obsoletes:      tmt-provision-bootc < %{version}-%{release}
+%endif
+Requires:       tmt == %{version}-%{release}
+Requires:       tmt+provision-virtual == %{version}-%{release}
+Requires:       podman
+# virtiofsd is for podman-machine
+Recommends:     virtiofsd
+
+%description -n tmt+provision-bootc %_metapackage_description
+
```

Comment:
At least a minimal `%files` section is also needed to convince `rpm-build` to actually create the subpackage. See bottom of the spec.